### PR TITLE
Add ishanSrt's report

### PR DESCRIPTION
**Pull request by ishanSrt**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: _reports/ishanSrt.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+  - 7:
+      - repo: coala-quickstart
+      - link: https://github.com/coala/coala-quickstart/commit/9e91285ac87cd36ee846743f1c9d891c324f8c18
+      - details: >
+         bear_settings.yaml: Fix wrong bear for settings
+
+  - 8:
+      - repo: coala-quickstart
+      - link: Project.py: Fix bug while printing languages
```
Comment (jayvdb) [inline _reports/ishanSrt.md]: this is the cause of the Netlify breakage.  It doesnt like the ":" , but a link here will solve that.

File: _reports/ishanSrt.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  - 10:
+      - repo: coala-quickstart
+      - link: https://github.com/coala/coala-quickstart/commit/94e9eda139234e8b9183ce5462bd7c9a9d97e197
+      - details: >
+         Aggregate green mode per file results
+
+
+---
+
+### Meta-review
```
Comment (jayvdb) [inline _reports/ishanSrt.md]: Green Mode

File: _reports/ishanSrt.md (hunk 1 above)
Comment (ishanSrt) [reply _reports/ishanSrt.md]: link? what link?

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/641/commits/178141aed20119700f99718d5b672408084b5b2e

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**